tasks/src/text-to-image/about.md

```diff
@@ -0,0 +1,23 @@
+## Use Cases
+
+### Data Generation
+  
+Businesses can create additional data for their task specific to their use cases by inputting text and getting image outputs. 
+  
+### Immersive Conversational Chatbots
+  
+Chatbots can be made more immersive if they provide contextual images based on the input provided by the user.
+  
+### Creative Ideas for Fashion Industry
+  
+Diverse design patterns can be generated to obtain unique fashion. This will make it easy for designers to conceptualize their design before actually implementing it.
+  
+### Architecture Industry
+  
+Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.
+  
+## Useful Resources
+- [MinImagen - Build Your Own Imagen Text-to-Image Model](https://www.assemblyai.com/blog/minimagen-build-your-own-imagen-text-to-image-model/)
+- [OpenAI Blog - Dall E](https://openai.com/blog/dall-e/)
+- [Text to Image](https://towardsdatascience.com/text-to-image-a3b201b003ae)
```

Comment:
I'd rather remove personal blogs but keep blog from openai & assembly as they're subjected to technical writing checks & more 🙂

Reply:
I have removed the personal blogs from the list.